src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -428,6 +460,10 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 if(defaultRecTypeID != null)
                     a.RecordTypeID = defaultRecTypeID;
 
+                if(Advancement_Info.useAdv() && Advancement_Info.getApiHEDA() != null) {
+                    Advancement_Info.getApiHEDA().configAccountForContact(a);
+                }
+
```

Comment:
Are there unit tests for the changes made in this class in The Advancement repo? Otherwise, is there some other way to verify this code is correct and be alerted if it breaks?

Reply:
That is a good point. I think the logic in HEDA is empty. So, if HEDA unit test should cover it. If the logic changes, HEDA unit test will break. 
But we do need to add unit test in Adv repo to see if the injected logic works. 
What do you think?

Reply:
Yes, I think we should have unit tests in the ADV repo for it. Maybe there are tests in NPSP that we can reuse for this with minor changes?